src/components/ConnectWallet/ConnectWalletDialog.tsx

```diff
@@ -0,0 +1,154 @@
+import { Trans } from '@lingui/macro'
+import METAMASK_ICON_URL from 'assets/images/metamaskIcon.png'
+import WALLETCONNECT_ICON_URL from 'assets/images/walletConnectIcon.svg'
+import Button from 'components/Button'
+import Column from 'components/Column'
+import { Header } from 'components/Dialog'
+import Row from 'components/Row'
+import { Web3Context, Web3ContextType } from 'hooks/connectWeb3/useActiveWeb3React'
+import useConnect, { connections, Web3Connection } from 'hooks/connectWeb3/useConnect'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+const Content = styled(Column)``
+const Heading = styled(Column)``
+const Footing = styled(Column)``
```

Comment:
it looks like these components are not used. Is that right? (same for associated styles below)